recipes/default.rb

```diff
@@ -33,7 +33,13 @@
   haproxy_lb "admin" do
     bind "#{admin['address_bind']}:#{admin['port']}"
     mode 'http'
-    params({ 'stats' => 'uri /'})
+    params(
+      'stats' => <<-STATS
+uri /
```

Comment:
👏 so so nice to actually fix this. Having the no-downtime reload in place actually allows us to now do things with haproxy. Exciting!